hw/hw.md

```diff
@@ -0,0 +1,38 @@
+# Hardware
+
+The NervesKey is a tiny circuit board with an ATECC608A cryptographic chip from
+Microchip Technology. It is an inexpensive addition to the Raspberry Pi and
```

Comment:
Is it worth linking to `Microchip Technology`? I’m just assuming this is a company or brand

Reply:
Fixed.